components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -598,18 +540,22 @@ void ConversationHandler::SendFeedback(const std::string& category,
       },
       std::move(callback));
 
-  if (!associated_content_delegate_) {
-    send_hostname = false;
+  std::vector<std::string> urls;
+  if (send_hostname) {
+    for (const auto& content :
+         associated_content_manager_->GetAssociatedContent()) {
+      if (!content->url.is_valid() || !content->url.SchemeIsHTTPOrHTTPS()) {
+        continue;
+      }
+      urls.push_back(content->url.host());
+    }
   }
 
-  const GURL page_url =
-      send_hostname ? associated_content_delegate_->GetURL() : GURL();
-
-  feedback_api_->SendFeedback(category, feedback, rating_id,
-                              (send_hostname && page_url.SchemeIsHTTPOrHTTPS())
-                                  ? std::optional<std::string>(page_url.host())
-                                  : std::nullopt,
-                              selected_language_, std::move(on_complete));
+  feedback_api_->SendFeedback(
+      category, feedback, rating_id,
+      urls.empty() ? std::nullopt
```

Comment:
@porteron @Steeeephen is this the way you want the feedback API to receive multiple hostnames for multiple associated content of a single conversation (comma-delimited)?

Reply:
The api expects a string so passing a comma delimited list of urls should be fine.

Reply:
Phew! Hadn't even considered